.changeset/popular-adults-behave.md

```diff
@@ -0,0 +1,6 @@
+---
+'@guardian/source-foundations': patch
+---
+
+- fix invalid CSS syntax in the exported reset.
+- add assertions and new unit tests for areas where we export CSS and want to ensure that SCSS does not slip into the code.
```

Comment:
as this is an internal implementation detail, i'd be inclined to remove this. knowing that tests have been added doesn't affect how i would consume the package, so we maybe don't need to add this to the changelog?

```suggestion
- add assertions and new unit tests for areas where we export CSS and want to ensure that SCSS does not slip into the code.
```

Reply:
good point 👍